installers/msi-language/ActiveState/Rollbar.cs

```diff
@@ -118,13 +118,20 @@ private static void Report(Level level, string message, Session session, IDictio
 	    
             if (!criticalReported)
             {
-                if (level == Level.Critical)
+                try
                 {
-                    criticalReported = true;
-                    RollbarLocator.RollbarInstance.AsBlockingLogger(RollbarTimeout).Critical(new GenericException(message), customFields);
-                } else
+                    if (level == Level.Critical)
+                    {
+                        criticalReported = true;
+                        RollbarLocator.RollbarInstance.AsBlockingLogger(RollbarTimeout).Critical(new GenericException(message), customFields);
+                    }
+                    else
+                    {
+                        RollbarLocator.RollbarInstance.AsBlockingLogger(RollbarTimeout).Error(new GenericException(message), customFields);
+                    }
+                } catch (System.Exception e)
                 {
-                    RollbarLocator.RollbarInstance.AsBlockingLogger(RollbarTimeout).Error(new GenericException(message), customFields);
+                    session.Log("Logging to rollbar failed with error: {0}", e);
```

Comment:
Would it be worthwhile trying to send an S3/GA Event in this case? That we we can potentially get some information when this is happening.

Reply:
That is a good idea.  @Naatan can I just decide on the event name?  I would choose category "error", label "rollbar".

Reply:
Sounds good